adapters/onetag/onetag.go

```diff
@@ -0,0 +1,169 @@
+package onetag
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type onetagAdapter struct {
```

Comment:
There's no reason to repeat the package name. Please prefer the simpler name `adapter`.

Reply:
I renamed it to adapter!